contracts/contracts/governance/Strategizable.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity ^0.8.0;
+
+import { Governable } from "./Governable.sol";
+
+contract Strategizable is Governable {
+    event StrategistUpdated(address _address);
+
+    // Address of strategist
+    address public strategistAddr;
```

Comment:
Let's put some _gap space in this contract, just in case. Good habit to have for inheritable contracts with storage slots.

Reply:
Done